packages/wct-mocha/src/index.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * @license
+ * Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+/**
+ * This file is the entry point into `web-component-tester`'s browser client.
+ */
+// Registers a bunch of globals:
+import './environment/helpers.js';
+
+import ChildRunner from './childrunner.js';
+import CLISocket from './clisocket.js';
+import * as config from './config.js';
+import * as environment from './environment.js';
+import * as errors from './environment/errors.js';
+import * as mocha from './mocha.js';
+import * as reporters from './reporters.js';
+import MultiReporter from './reporters/multi.js';
+import * as suites from './suites.js';
+import * as util from './util.js';
+
+export {extendInterfaces} from './mocha/extend.js';
+export {ChildRunner};
+
+declare global {
+  interface Window {
+    __useNpm: boolean;
+    define: Function;
+  }
+}
+
+export function initialize(initConfig?: config.Config) {
+  const _config = {};
+
+  if (initConfig) {
+    config.deepMerge(_config, initConfig);
+  }
+
+  // You can configure WCT before it has loaded by assigning your custom
+  // configuration to the global `WCT`.
+  if (window.WCT) {
+    config.deepMerge(_config, window.WCT as {} as config.Config);
+  }
+
+  config.setup(_config);
+
+  // Maybe some day we'll expose WCT as a module to whatever module registry you
+  // are using (aka the UMD approach), or as an es6 module.
```

Comment:
update comment?

Reply:
Too costly to work out how to get wct-mocha to be useful as an ES6 module.

Reply:
If mocha was module-based I'd say *heck yeah* but its just not worth teasing these things apart in this iteration.